builder/ecs/access_config.go

```diff
@@ -82,16 +86,27 @@ func (c *AlicloudAccessConfig) Client() (*ClientWrapper, error) {
 		_ = endpoints.AddEndpointMapping(c.AlicloudRegion, "Ecs", c.CustomEndpointEcs)
 	}
 
+	if c.AlicloudAccessKey == "" || c.AlicloudSecretKey == "" {
+		c.AlicloudAccessKey = getProviderConfig(c.AlicloudAccessKey, "access_key_id")
+		c.AlicloudSecretKey = getProviderConfig(c.AlicloudSecretKey, "access_key_secret")
+	}
+
+	if c.AlicloudRamRoleArn == "" || c.AlicloudRamSessionName == "" {
+		c.AlicloudRamRoleArn = getProviderConfig(c.AlicloudRamRole, "ram_role_arn")
+		c.AlicloudRamSessionName = getProviderConfig(c.AlicloudRamSessionName, "ram_session_name")
+	}
+
 	if c.AlicloudRamRole == "" {
 		c.AlicloudRamRole = getProviderConfig(c.AlicloudRamRole, "ram_role_name")
 	}
-	if c.AlicloudRamRole != "" {
+
```

Comment:
put AlicloudRamRole first may be better.

Reply:
updated